ibm/service/power/data_source_ibm_pi_volume_group_remote_copy_replationships_test.go

```diff
@@ -0,0 +1,37 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
```

Comment:
Could you please fix the typo in the file name?

Reply:
done